tests/apps/pubsub-publisher/app.go

```diff
@@ -107,13 +108,22 @@ func getBulkRequetMetadata(r *http.Request) map[string]string {
 	return metadata
 }
 
+func getBulkPublishPubsubOrDefault(reqMeta map[string]string) string {
+	if v, ok := reqMeta[bulkPubsubMetaKey]; ok {
+		delete(reqMeta, bulkPubsubMetaKey)
+		return v
+	}
+	return pubsubName
+}
```

Comment:
This function is doing something different than its name, no?

Reply:
Can be changed to getAndDeleteBulkPubsubnameOrDefault....felt that the name was too long...

Will add a comment for the function as well as where it is being called that the metadata for the test is being deleted